src/components/Utils/queries.js

```diff
@@ -446,3 +446,31 @@ export const getAccessibleResources = async (
   });
   return { accessibleFieldObject, unaccessibleFieldObject };
 };
+
+export const guppySinglefieldSearch = (path, type, keyword, field, numResults) => {
```

Comment:
does this function get used anywhere?

Reply:
Oof, good catch, no it doesn't -- it was originally used here but I refactored it out: https://github.com/uc-cdis/guppy/blob/58913a8a569384d7cb5776d18c00d7dd7021297b/src/components/ConnectedFilter/utils.js#L63-L64

I'll remove it now!